pom.xml

```diff
@@ -1023,8 +1023,8 @@
             </dependency>
           </dependencies>
           <configuration>
-            <configLocation>beam/checkstyle.xml</configLocation>
```

Comment:
@wtanaka, could this be a versioning issue, i.e., Travis uses an older version of something so we don't see this problem elsewhere?